### PR TITLE
Cleanups

### PR DESCRIPTION
**Pull request by paul-dingemans**
## Description

Based on the IntelliJ Code inspections I have cleaned up some minor problems in the code base. Different problems are commited separately. So it is best to review this per commit.

## Checklist

- [ ] tests are added
- [ ] `CHANGELOG.md` is updated

### CODE REVIEW

**Review by Goooler — APPROVED** [review]

**Comment by Goooler** [comment]
You can use squash commit on this PR.

**Comment by paul-dingemans** [comment]
Hi @Goooler 
Tnx for reviewing. It would be really cool if you have time for review of other PR's as well. The other maintainers are quite busy and have little time for this project. I would appreciate an independent review before merging my changes. For example #1380 ;-)

**Comment by paul-dingemans** [comment]
> You can use squash commit on this PR.

Tnx for the advise. I have commited them sepately wit the purpose to merge wihout squashing. The changes are not related and squashing them would make it to one ball of mud. Normally these should have been split in separate PR's and be merged indidually. But that was just too much overhead ;-)

**Pull request merged**
